packages/web-pkg/tests/unit/composables/piniaStores/modals.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { getComposableWrapper } from 'web-test-helpers'
+import { useModals } from '../../../../src/composables/piniaStores'
+import { createPinia, setActivePinia } from 'pinia'
+
+describe('useModals', () => {
+  beforeEach(() => {
+    setActivePinia(createPinia())
+  })
+
+  describe('method "dispatchModal"', () => {
+    it('adds a modal to the stack of modals', () => {
+      getWrapper({
+        setup: (instance) => {
+          const data = { title: 'test' }
+          const modal = instance.dispatchModal(data)
+
+          expect(modal.id).toBeDefined()
+          expect(modal.title).toEqual(data.title)
+          expect(instance.activeModal).toEqual(modal)
```

Comment:
Would be nice to add a second one and then check again the right modals are (in)active.

Same for removeModal